web/src/components/RegistCheck.tsx

```diff
@@ -0,0 +1,70 @@
+// Checking console.log
+export const logConsole = (text1: string, bool?: boolean) => {
+  console.log(text1, bool);
+};
+
+// If Email input data is Empty
+export const isEmailEmpty = (email: string) => {
+  if (email === '') return true;
+  return false;
+};
+
+// If Email input data meets the email regular expression
+export const CheckEmail = (email: string) => {
+  const emailRegex = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i;
+  return emailRegex.test(email);
+};
+
+// If Password input data is Empty
+export const isPasswordEmpty = (password: string) => {
+  if (password === '') return true;
+  return false;
+};
+
+// If Password input data meets the Password regular expression
+export const CheckPassword = (password: string) => {
+  const PasswordRegex = /^(?=.*[a-zA-Z])(?=.*[!@#$%^*+=-])(?=.*[0-9]).{8,25}$/;
+  return PasswordRegex.test(password);
+};
+
+// If Name input data is Empty
+export const isNameEmpty = (name: string) => {
+  if (name === '') return true;
+  return false;
+};
+
+// If PhoneNumber input data is Empty
+export const isPhoneNumberEmpty = (phoneNumber: string) => {
+  if (phoneNumber === '') return true;
+  return false;
+};
+
+// If PhoneNumber input data meets the PhoneNumber regular expression
+export const CheckPhNumber = (phnumber: string) => {
```

Comment:
Please rename `CheckPhNumber` function. Some might think this is `Oh. So this is checking someone's student ID who's in Ph.D course?`